app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb

```diff
@@ -488,10 +488,11 @@ def hardware_disks(persister_hardware, disks)
           :location        => index.to_s,
           :size            => device.provisioned_size.to_i,
           :size_on_disk    => device.actual_size.to_i,
-          :disk_type       => device.sparse == true ? 'thin' : 'thick',
```

Comment:
Let's continue to set `:disk_type` until we remove the column and either update the UI or provide a compatibility method for them to call.

Reply:
done